### PR TITLE
Add guide for monitors within a range

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Add a guide for users that want to be alerted for a value within a range

### Motivation
Saw a couple related questions and figured it would be worth a guide. 

### Preview

https://docs-staging.datadoghq.com/nils/monitor-range/monitors/guide/
https://docs-staging.datadoghq.com/nils/monitor-range/monitors/guide/monitor-for-value-within-a-range

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Monitor if a value is inside or outside a range
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:2]:
```suggestion
title: Monitoring Ranges
```

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+kind: guide
+---
+
+## Overview
+
+If monitors supports to alert when a given value is above or below a given threshold, it is also possible to be notified if a given value is inside or outside of a given range.
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:8]:
```suggestion
If a monitor supports alerting when a given value is above or below a given threshold, it is possible to be notified if a given value is inside or outside of a range.
```

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### Metric Example
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:10]:
```suggestion
## Examples
### Metric
```

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Let's imagine that a metric `a` reports discrete values from `0` to `10` representing a status and you want to be notified if the metric is not between `4` and `8`.
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:12]:
```suggestion
Metric `a` reports discrete values from `0` to `10` representing a status and you want to be notified if the metric is not between `4` and `8`.
```

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Mathematically, this is the same as saying that the difference between the metric and the center of the range (6) should never be more than 2.
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:13]:
```suggestion
Mathematically, the difference between the metric and the center of the range (6) should never be more than 2. 
```

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+## Overview
+
+If monitors supports to alert when a given value is above or below a given threshold, it is also possible to be notified if a given value is inside or outside of a given range.
+
+### Metric Example
+
+Let's imagine that a metric `a` reports discrete values from `0` to `10` representing a status and you want to be notified if the metric is not between `4` and `8`.
+Mathematically, this is the same as saying that the difference between the metric and the center of the range (6) should never be more than 2. 
+
+`8 > a > 4` <=> `abs(6-a) < 2` <=> `abs(6-a) - 2 < 0`
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:15]: I recommend putting this in a code block

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+- To be notified if the value is outside the range, the monitor condition should be `abs(6-a) - 2 > 0`.
+- To be notified if the value is inside the range, the monitor condition should be `2 - abs(6-a) > 0`.
+
+{{< img src="monitors/faq/monitor_range.png" alt="metric monitor on a range"  >}}
+
+### Theoretical Example
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:22]:
```suggestion
### Theoretical
```

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+`8 > a > 4` <=> `abs(6-a) < 2` <=> `abs(6-a) - 2 < 0`
+
+- To be notified if the value is outside the range, the monitor condition should be `abs(6-a) - 2 > 0`.
+- To be notified if the value is inside the range, the monitor condition should be `2 - abs(6-a) > 0`.
+
+{{< img src="monitors/faq/monitor_range.png" alt="metric monitor on a range"  >}}
+
+### Theoretical Example 
+
+Let's assume the range is defined by `x > a > y` with `a` the metric in question.
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:24]:
```suggestion
A range is defined by `x > a > y` with `a` being the metric in question. 
```

File: content/en/monitors/guide/monitor-for-value-within-a-range.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+{{< img src="monitors/faq/monitor_range.png" alt="metric monitor on a range"  >}}
+
+### Theoretical Example 
+
+Let's assume the range is defined by `x > a > y` with `a` the metric in question. 
+
+- To be notified if the value is inside the range, the monitor condition should be: `abs((x-y/2) - a) - (x-y)/2 > 0`.
+- To be notified if the value is outside the range, the monitor condition should be: `(x-y)/2 - abs((x-y/2) - a) > 0`.
+
+[Reach out to the Datadog support team][1] if you have any questions.
```
Comment (ruthnaebeck) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:29]:
```suggestion
## Troubleshooting

Need help? Contact [Datadog support][1].
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

File: content/en/monitors/guide/monitor-for-value-within-a-range.md (hunk 6 above)
Comment (NBParis) [inline content/en/monitors/guide/monitor-for-value-within-a-range.md:16]:
```suggestion
```
8 > a > 4 <=> abs(6-a) < 2 <=> abs(6-a) - 2 < 0
```
```

Comment (NBParis) [reply content/en/monitors/guide/monitor-for-value-within-a-range.md]: done :)

**Review by ruthnaebeck — APPROVED** [review]
👍

**Pull request merged**